viam-cartographer/src/slam_service/slam_service.cc

```diff
@@ -158,6 +158,141 @@ ::grpc::Status SLAMServiceImpl::GetInternalStateStream(
     return grpc::Status::OK;
 }
 
+::grpc::Status SLAMServiceImpl::GetPosition(ServerContext *context,
```

Comment:
[note to self] copy-pasted the implementations from the corresponding temp methods